pip/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
-name = "qsharp"
-version = "0.0.0"
+name = "qsharp_python"
```

Comment:
Feel free to propose a better name instead of "qsharp_python".
For now I named it such that it is consistent with the name in "pip/Cargo.toml".